.github/workflows/pr-fuzz.yaml

```diff
@@ -24,9 +24,17 @@ jobs:
         fuzz-seconds: 600
         dry-run: false
         language: c
+        output-sarif: true
     - name: Upload Crash
       uses: actions/upload-artifact@v3
       if: failure() && steps.build.outcome == 'success'
       with:
         name: artifacts
         path: ./out/artifacts
+    - name: Upload Sarif
+      if: always() && steps.build.outcome == 'success'
+      uses: github/codeql-action/upload-sarif@v2
+      with:
+        # Path to SARIF file relative to the root of the repository
+        sarif_file: cifuzz-sarif/results.sarif
```

Comment:
I think we should add `category: fuzzing` or similar here?

Reply:
@jonathanmetzman I was thinking about the same thing when going through the docs: https://docs.github.com/en/code-security/code-scanning/integrating-with-code-scanning/uploading-a-sarif-file-to-github#example-workflow-for-sarif-files-generated-outside-of-a-repository -- what's your take on this in the context of CIFuzz?

Reply:
I think it's primarily to add more context to the specific result (there may be lots), I did this for the regular scans of the container for example: https://github.com/fluent/fluent-bit/blob/b266d46e5851651f34889a1430491764ed62a074/.github/workflows/cron-trivy.yaml#L78

That way you know where a result comes from when we have potentially lots of results from disparate tools, e.g. code scanning, fuzzing, container CVEs, etc.

Reply:
Sounds good, thanks @patrick-stephens -- I added it!